tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/record/OpenRecordTest.java

```diff
@@ -89,6 +89,12 @@ public void testDefaultValue() {
 
         Assert.assertTrue(returns.get(2) instanceof Long);
         Assert.assertEquals(returns.get(2), 999L);
+
+        Assert.assertTrue(returns.get(3) instanceof Long);
+        Assert.assertEquals(returns.get(3), 10L);
+
+        Assert.assertTrue(returns.get(4) instanceof Long);
+        Assert.assertEquals(returns.get(4), 1L);
```

Comment:
If we are continuing with this test, let's move the assertion for just this test to Ballerina.

Reply:
Fixed